packages/phpstan-rules/build/build-preload.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
not sure, if/when/whether this preload.php script is usefull for the phpstan-rules case. copied it over from `ecs` so we can discuss what needs to be done.